CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.0)
 
 project(nibbler)
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
You need to sort out your machine to use 17. All updated versions of gcc and clang support it.